### ISSUE TITLE
Unread message count Badge doesn't reset to 0 if there is no unread messages left

### ISSUE THREAD

**Issue by maulikvora** [RocketChat/Rocket.Chat#11688]
### Description:

Unread message count stuck at 1 despite being no unread message.

### Steps to reproduce:

1. When someone sends me message, unread counts increases
2. now read that message
3. Unread count badge remains there as 1 in bottom dock

### Expected behavior:

Unread message count on Rocketchat icon on Dock should be reset to 0.

### Actual behavior:
  Counts in Top macbar is being set to 0, just count-badge in bottom dock on Rocketchat icon remains 1.

### Server Setup Information:

- Version of Rocket.Chat Server: 0.66.3
- Operating System: Linux
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v8.11.3
- MongoDB Version:

**Comment by thirsch** [comment]
For me, this issue started with version 2.12.0 of the client.

**Comment by maulikvora** [comment]
yes right @thirsch it started appearing with latest updated 2-3 days back

**Comment by kaiiiiiiiii** [comment]
Wrong repository and also a duplicate [#801](https://github.com/RocketChat/Rocket.Chat.Electron/issues/801).

// CC: @TwizzyDizzy
